### PR TITLE
Feature zipfile

### PR DESCRIPTION
**Pull request by Raymundogs**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/controller/AudioConverterController.java [hunk 1]
```diff
@@ -9,14 +9,18 @@
 package org.fundacion.jala.converter.controller;
 
 import org.fundacion.jala.converter.service.AudioConverter;
+import org.fundacion.jala.converter.service.ExtractMetadata;
 import org.fundacion.jala.converter.service.FileStorageService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
 import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/controller/AudioConverterController.java:20]: Delete blank line

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java [hunk 2]
```diff
@@ -18,11 +18,31 @@
...
+    private final String FOUR_POINT_ZERO_CHANNEL = "4";
+    private final String FIVE_POINT_ZERO_CHANNEL = "5";
+    private final String FIVE_POINT_ONE_CHANNEL = "6";
+    private final String SIX_POINT_ONE_CHANNEL = "7";
+    private final String SEVEN_POINT_ONE_CHANNEL = "8";
+
+    public AudioConverter() {
+    }
+
+    public AudioConverter(String format, String bitrate, String hz, String volume, String audioChannels) {
```
Comment (GustavoZHA) [inline src/main/java/org/fundacion/jala/converter/service/AudioConverter.java]: The parameter would be final agree checkstyle

**Review by GustavoZHA — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/service/AudioConverter.java (hunk 2 above)
Comment (Raymundogs) [reply src/main/java/org/fundacion/jala/converter/service/AudioConverter.java]: solved

File: src/main/java/org/fundacion/jala/converter/controller/AudioConverterController.java (hunk 1 above)
Comment (Raymundogs) [reply src/main/java/org/fundacion/jala/converter/controller/AudioConverterController.java:20]: solved

**Review by edsonarios — APPROVED** [review]
Good job

File: src/main/java/org/fundacion/jala/converter/service/RunCommand.java [hunk 3]
```diff
@@ -26,9 +26,14 @@
      * @param command
      */
     public void run(final String command) {
+        boolean isWindows = System.getProperty("os.name")
+                .toLowerCase().startsWith("windows");
         ProcessBuilder processBuilder = new ProcessBuilder();
-
-        processBuilder.command("bash", "-c", command);
+        if (isWindows) {
+            processBuilder.command("cmd.exe", "/c", command);
```
Comment (SaulCM) [inline src/main/java/org/fundacion/jala/converter/service/RunCommand.java:33]: cmd and bash could be final :)

**Review by SaulCM — APPROVED** [review]

**Pull request merged**